src/app/zap-templates/zcl/data-model/chip/thread-border-router-management-cluster.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0452"/>
+    <field name="PANChange" mask="0x1"/>
```

Comment:
Spec has a space in there.  Spec should be fixed to not have that space....  Please make sure a spec issue is filed.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/9280 Filed the issue

Reply:
Opened PR https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/9282 to fix in spec. The space can remain absent.